man/Makefile.am

```diff
@@ -62,53 +62,38 @@ man7_MANS = $(MAN7)
 	$(A2X) --doctype manpage --format manpage $<
 	$(A2X) --doctype manpage --format xhtml $<
 
-# Man pages built byt a2x based on the NAMES section of the .txt file
+# Man pages built byt a2x based on the NAMES section of the .txt file.
+# Note - this list includes all the defined entries, but a2x only builds the first 10.
```

Comment:
Why not trimming the list to ten entries to avoid the warning?

```
shell for ... done | sed 10q
```

Reply:
It is a2x generating the warning when it parses the individual .txt file.  I wanted to have the full list of names show in the man page (and doxgen output), and  A2X_EXTRA_PAGES is just used to install all the man pages (not used by a2x).  Use of A2X_EXTRA_PAGES in the install provides a safety check that all the .3 files that are in the NAMES section have been correctly built.

Reply:
Okay.